### PR TITLE
Mas 2.2.00

### PR DESCRIPTION
**Pull request by ysd24**

### CODE REVIEW

**Review by zakirjt — APPROVED** [review]
looks fine

File: README.md [hunk 1]
```diff
@@ -315,8 +315,8 @@ This software may be modified and distributed under the terms
 of the MIT license. See the [LICENSE][license-link] file for details.
 
  [mag]: https://docops.ca.com/mag
- [get-started]: http://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/mobile-sdk-for-ca-mobile-api-gateway/2-0.html
- [docs]: http://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/mobile-sdk-for-ca-mobile-api-gateway/2-0.html
+ [get-started]: http://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/mobile-sdk-for-ca-mobile-api-gateway/2-2.html
```
Comment (zakirjt) [inline README.md:318]: Please check these links, they are not working.

**Review by zakirjt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zakirjt) [reply README.md:318]: As per @arindamhit , this links will be available after release, so please ignore the above.

**Review by zakirjt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zakirjt) [reply README.md:318]: As per @arindamhit , this links will be available after release, so please ignore the above.

**Review by as666499 — APPROVED** [review]
LGTM

**Pull request merged**